api/gr.js

```diff
@@ -54,7 +54,7 @@ const createEntity = (user, callback) => {
     }
   }, (e, r, body) => {
     if (e || r.statusCode >= 400) {
-      callback(false)
+      callback(false) // eslint-disable-line
```

Comment:
Why did this line need to be disabled for eslint?

Reply:
standardjs error, When you use the name `callback` or `cb` and don't pass and `Error` object as the first param, it gives an error. `standard/no-callback-literal`